vmware-base/src/com/cloud/hypervisor/vmware/util/VmwareContext.java

```diff
@@ -375,6 +377,18 @@ public void uploadFile(String urlString, File localFile) throws Exception {
         }
     }
 
+    private Charset getCharSetFromConnection(HttpURLConnection conn) {
```

Comment:
Hi Daan, I didnt see any usages for it except for VmwareContext. May be utility method to get character encoding from a URL would be useful than a connection. Can I take this up later?